src/server/pfs/server/driver.go

```diff
@@ -2194,8 +2194,11 @@ func (d *driver) getFile(ctx context.Context, file *pfs.File, offset int64, size
 	if err != nil {
 		return nil, err
 	}
+	if len(paths) <= 0 {
+		return nil, fmt.Errorf("no file(s) found that match %v", file.Path)
+	}
 
-	var objects []*pfs.Object
+	objects := []*pfs.Object{}
```

Comment:
Any reason you changed this? Doing it with `var` is more standard for our codebase. And it's slightly shorter, 25 characters vs 26, think of all the bits we'll save.